### PR TITLE
Node Proxy

### PR DESCRIPTION
**Pull request by burgwyn**
Simple node app to proxy the SAML2 metadata endpoint

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const path = require('path');
+
+// define the API root url
+const API_ROOT = process.env.API_ROOT || 'http://localhost:8000';
+
+// define the url consuming the API, provide local dev value as default
+const API_URL = process.env.API_URL || `${API_ROOT}/api/v1`;
```
Comment (mjoyce91) [inline src/server.js]: Is this variable ever used?

File: src/server.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+const port = process.env.PORT || 3000;
+
+const app = express();
+
+// middleware for static assets
+app.use(PUBLIC_URL, express.static(path.join(__dirname, '../build')));
+
+// saml2 metadata
+app.get(`${PUBLIC_URL}metadata`, (request, response) => {
+  response.redirect(`${API_ROOT}/saml2/metadata`);
```
Comment (mjoyce91) [inline src/server.js]: Maybe you meant to use URL here? Should the saml2 route have the `/api/v1` prefix?

**Comment by burgwyn** [comment]
The SAML and API pieces have separate routes, which was why I created to the two variables.  We're not proxying any of the API functions currently, so I removed it.

File: src/server.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+// application port
+const port = process.env.PORT || 3000;
+
+const app = express();
+
+// middleware for static assets
+app.use(PUBLIC_URL, express.static(path.join(__dirname, '../build')));
+
+// saml2 metadata
+app.get(`${PUBLIC_URL}metadata`, (request, response) => {
```
Comment (mjoyce91) [inline src/server.js]: The API usually requires trailing slashes. Is that needed here as well?

**Pull request merged**